Developer_Documentation/1.6/code_docs/modules/order_total_modules.md

```diff
@@ -0,0 +1,334 @@
+# Introduction
+Zen Cart *order_total* modules are used to calculate (and display) sub-totals, taxes, totals and other intermediary values for an order. *Order-total* modules can also calculate discounts or other subtractions from the amount a customer owes for an order. An *order-total* includes, at a minimum, two files:
+
```

Comment:
At the end of this line you say "An order-total includes ... two files".

I'm thinking it should probably say "An order-total **module** includes ... two files"